src/index.html

```diff
@@ -1,150 +1,139 @@
 <!DOCTYPE html>
-<html>
-
+<html lang="en">
 <head>
   <title>Activity Tracker</title>
   <!-- <link rel="stylesheet" href="normalize.css"> -->
-  <link rel="stylesheet" href="styles.css">
+  <!-- <link rel="stylesheet" href="styles.scss"> -->
   <link href="https://fonts.googleapis.com/css?family=Open+Sans:300,400,600,700&display=swap" rel="stylesheet">
-  <link href="https://fonts.googleapis.com/css?family=Raleway:400,300,600,800,900" rel="stylesheet" type="text/css">
+  <!-- <link href="https://fonts.googleapis.com/css?family=Raleway:400,300,600,800,900" rel="stylesheet" type="text/css"> -->
   <script src="https://cdn.jsdelivr.net/npm/chart.js@2.8.0"></script>
   <script src="https://cdn.rawgit.com/kimmobrunfeldt/progressbar.js/0.7.0/dist/progressbar.js"></script>
-
 </head>
-
 <body>
-  <section class="welcome-section">
```

Comment:
Nice work recognizing that this class isn't ever used in an effective way. This makes the HTML much more semantic.